metricbeat/module/elasticsearch/enrich/_meta/data.json

```diff
@@ -1,38 +1,57 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
-    "elasticsearch": {
-        "cluster": {
-            "id": "et6blfihSoytMUvkpYtEKQ",
-            "name": "docker-cluster"
-        },
-        "enrich": {
-            "executed_searches": {
```

Comment:
If I'm following this correctly, we are introducing a breaking change here, as we're going from `elasticsearch.enrich.executed_searches` to `elasticsearch.enrich.coordinator_stats.executed_searches`. Same applies to other fields as well. We should try to prevent this, maybe by introducing field aliases from the old to the new fields?

Reply:
*Bump*